src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -352,6 +352,34 @@ public async Task<DicomWebResponse<ChangeFeedEntry>> GetChangeFeedLatest(string
                 });
         }
 
+        public async Task<DicomWebResponse> AddCustomTagAsync(IEnumerable<CustomTagEntry> customTagEntries, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(customTagEntries, nameof(customTagEntries));
+            using var request = new HttpRequestMessage(
+                HttpMethod.Post,
+                new Uri(DicomWebConstants.BaseCustomTagUri, UriKind.Relative));
```

Comment:
Since `Uri` is immutable, let's save this as a static readonly field somewhere in the client or as a constant